src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2363,8 +2478,7 @@ export default class AircraftModel {
                     headingDisplay,
                     destinationDisplay,
                     isFollowingSid,
-                    isFollowingStar,
-                    fixRestrictions
+                    isFollowingStar
```

Comment:
If that file exists in your branch, please remove it and all the methods in `AircraftModel`.  I'm sure they stuck around because you had some changes there or some thing that then led to a bad merge.